pom.xml

```diff
@@ -389,7 +389,7 @@
 					<plugin>
 						<groupId>org.jacoco</groupId>
 						<artifactId>jacoco-maven-plugin</artifactId>
-						<version>0.7.9</version>
+						<version>0.8.2</version>
```

Comment:
maybe worth upgrading to 0.8.3 as it is the first version to "officially" support java 11 "JaCoCo now officially supports Java 11 (GitHub #760)." https://www.jacoco.org/jacoco/trunk/doc/changes.html

(note for future, it would be better to not have a version defined here and rely on the version in Jboss tools parent but seems they are really late 0.7.1 https://github.com/jbosstools/jbosstools-build/blob/master/parent/pom.xml#L183 )

Reply:
so do we want to update version in boss tools parent or only update our pom.xml to version 0.8.3, because our UI tests are failing because of it

Reply:
go to update in our codebase first.

Reply:
ok I will update PR to version 0.8.4.201903161048

Reply:
to 0.8.3, 0.8.4.201903161048 is a snapshot version, it will vanish in few days/weeks